spec/services/course/assessment/answer/programming_auto_grading_service_spec.rb

```diff
@@ -135,6 +139,18 @@
               expect(answer).to be_correct
               expect(answer.grade).to eq(question.maximum_grade)
             end
+
+            context 'when autograded assessment uses evaluation test for grade/exp assignment' do
+              before do
+                allow(answer.submission.assessment).to receive(:use_evaluation).and_return(true)
+                allow(answer.submission.assessment).to receive(:autograded?).and_return(true)
+                subject.save!
+              end
+              it 'deducts grade for the failed evaluation test cases' do
```

Comment:
Does hound  complain if there's a blank line above this? The other `it` blocks usually have a blank line before them.